src/components/DefaultSecondaryMasthead/DefaultSecondaryMasthead.tsx

```diff
@@ -17,7 +17,7 @@ export default class DefaultSecondaryMasthead extends React.Component {
     }
 
     if (path === 'jaeger') {
```

Comment:
Perhaps with the changes "path == 'jaeger'" won't never happen ? I think.
Unless I miss some cornercase here.

Reply:
you're right